Minimum PHP version?

In the `package.xml` file we [currently have](https://github.com/chrislim2888/IP2Location-PECL-Extension/blob/8.3.0/package.xml#L54) that the declared minimum PHP version is 5.4.0:

```xml
 <dependencies>
    <required>
      <php>
        <min>5.4.0</min>
      </php>
      <!-- [omissis] -->
    </required>
  </dependencies>
```

Is that true? I can't install the ip2location PHP extension with a PHP lower than 8.0: with previous versions we have the following compilation error:

```
downloading ip2location-8.3.0.tgz ...
Starting to download ip2location-8.3.0.tgz (281,883 bytes)
..........................................................done: 281,883 bytes
8 source files, building
running: phpize
Configuring for:
PHP Api Version:         20190902
Zend Module Api No:      20190902
Zend Extension Api No:   320190902
building in /tmp/pear/temp/pear-build-defaultuser2VQ7I9/ip2location-8.3.0
running: /tmp/pear/temp/ip2location/configure --with-php-config=/usr/local/bin/php-config
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for PHP prefix... /usr/local
checking for PHP includes... -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/lib/php/extensions/no-debug-non-zts-20190902
checking for PHP installed headers prefix... /usr/local/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for gawk... no
checking for nawk... nawk
checking if nawk is broken... no
checking for ip2location support... yes, shared
checking for ip2location files in default path... found in /usr/local
checking for IP2Location_set_lookup_mode in -lIP2Location... yes
checking for a sed that does not truncate output... /bin/sed
checking for ld used by cc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for /usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 1572864
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if cc supports -fno-rtti -fno-exceptions... no
checking for cc option to produce PIC... -fPIC
checking if cc PIC flag -fPIC works... yes
checking if cc static flag -static works... yes
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
configure: patching config.h.in
configure: creating ./config.status
config.status: creating config.h
running: make -j20
/bin/bash /tmp/pear/temp/pear-build-defaultuser2VQ7I9/ip2location-8.3.0/libtool --mode=compile cc  -I. -I/tmp/pear/temp/ip2location -DPHP_ATOM_INC -I/tmp/pear/temp/pear-build-defaultuser2VQ7I9/ip2location-8.3.0/include -I/tmp/pear/temp/pear-build-defaultuser2VQ7I9/ip2location-8.3.0/main -I/tmp/pear/temp/ip2location -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/usr/local/include  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/temp/ip2location/ip2location.c -o ip2location.lo
mkdir .libs
 cc -I. -I/tmp/pear/temp/ip2location -DPHP_ATOM_INC -I/tmp/pear/temp/pear-build-defaultuser2VQ7I9/ip2location-8.3.0/include -I/tmp/pear/temp/pear-build-defaultuser2VQ7I9/ip2location-8.3.0/main -I/tmp/pear/temp/ip2location -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/usr/local/include -DHAVE_CONFIG_H -g -O2 -c /tmp/pear/temp/ip2location/ip2location.c  -fPIC -DPIC -o .libs/ip2location.o
In file included from /usr/local/include/php/main/php.h:37,
                 from /tmp/pear/temp/ip2location/ip2location.c:22:
/tmp/pear/temp/ip2location/ip2location_legacy_arginfo.h:83:37: error: 'arginfo_ip2location_get_asn' undeclared here (not in a function); did you mean 'zif_ip2location_get_asn'?
   83 | #define arginfo_ip2location_get_asn arginfo_ip2location_get_asn
      |                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/include/php/Zend/zend_API.h:67:75: note: in definition of macro 'ZEND_FENTRY'
   67 | #define ZEND_FENTRY(zend_name, name, arg_info, flags) { #zend_name, name, arg_info, (uint32_t) (sizeof(arg_info)/sizeof(struct _zend_internal_arg_info)-1), flags },
      |                                                                           ^~~~~~~~
/tmp/pear/temp/ip2location/ip2location_legacy_arginfo.h:187:2: note: in expansion of macro 'ZEND_FE'
  187 |  ZEND_FE(ip2location_get_asn, arginfo_ip2location_get_asn)
      |  ^~~~~~~
/tmp/pear/temp/ip2location/ip2location_legacy_arginfo.h:187:31: note: in expansion of macro 'arginfo_ip2location_get_asn'
  187 |  ZEND_FE(ip2location_get_asn, arginfo_ip2location_get_asn)
      |                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/tmp/pear/temp/ip2location/ip2location_legacy_arginfo.h:87:36: error: 'arginfo_ip2location_get_as' undeclared here (not in a function); did you mean 'zif_ip2location_get_as'?
   87 | #define arginfo_ip2location_get_as arginfo_ip2location_get_as
      |                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/include/php/Zend/zend_API.h:67:75: note: in definition of macro 'ZEND_FENTRY'
   67 | #define ZEND_FENTRY(zend_name, name, arg_info, flags) { #zend_name, name, arg_info, (uint32_t) (sizeof(arg_info)/sizeof(struct _zend_internal_arg_info)-1), flags },
      |                                                                           ^~~~~~~~
/tmp/pear/temp/ip2location/ip2location_legacy_arginfo.h:190:2: note: in expansion of macro 'ZEND_FE'
  190 |  ZEND_FE(ip2location_get_as, arginfo_ip2location_get_as)
      |  ^~~~~~~
/tmp/pear/temp/ip2location/ip2location_legacy_arginfo.h:190:30: note: in expansion of macro 'arginfo_ip2location_get_as'
  190 |  ZEND_FE(ip2location_get_as, arginfo_ip2location_get_as)
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
make: *** [Makefile:192: ip2location.lo] Error 1
ERROR: `make -j20' failed
```

--- a/package.xml
+++ b/package.xml
@@ -51,7 +51,7 @@
   <dependencies>
     <required>
       <php>
-        <min>5.4.0</min>
+        <min>8.0.0</min>
       </php>
       <pearinstaller>
         <min>1.4.0b1</min>

